src/teleop/gui/src/components/AutonModeCheckbox.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="wrap-button">
+    <button v-bind:class="[this.color]" v-on:click='toggleAndEmit()'>
+      <span class='white-text'>{{name}}: <br>{{active ? '\u2611' : '\u2610'}}</span>
+    </button>
+  </div>
+</template>
```

Comment:
I know this was called AutonModeCheckbox when we made it last year but it should probably just be more generally named, like TriStateCheckbox or something like that. Nothing inherent about this specifically for auton I don't think

Reply:
The other reason it is differentiated is that it is bigger due to some CSS in the component, probably a silly way of doing it though, but imo we should make an issues for it and a member can change the regular checkbox component in the future to be more abstract.

Reply:
Sure that sounds fine to me